runtime/ftplugin/astro.vim

```diff
@@ -0,0 +1,106 @@
+" Vim filetype plugin file
+" Language:     Astro
+" Maintainer:   Romain Lafourcade <romainlafourcade@gmail.com>
+" Last Change:  2022 Dec 5
+
+if exists("b:did_ftplugin")
+    finish
+endif
+let b:did_ftplugin = 1
+
+let s:cpo_save = &cpo
+set cpo-=C
+
+let b:undo_ftplugin = "iskeyword"
```

Comment:
The `b:undo_ftplugin` needs a complete command to undo. So `:let b:undo_ftplugin = 'setl iskeyword< '

Reply:
No, @chrisbra , `iskeyword<` is already used below. This looks like the result of a hasty substitution to me.

@pheiduck , that line should be:

```suggestion
let b:undo_ftplugin = "setlocal"
```
like in the original.